auth-web/src/models/Task.ts

```diff
@@ -0,0 +1,19 @@
+
+export interface Tasks {
+    task : Task[]
+}
+
+export interface Task {
+    created?: string;
+    createdBy?: string;
+    dateSubmitted?: string;
+    id?: number;
+    modified?: string;
```

Comment:
please add dueDate as an optional field. This is currently not used in orgs/products but for future